src/test/java/utils/seleniumWrapper.java

```diff
@@ -0,0 +1,35 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public class seleniumWrapper {
+
+    WebDriver driver = new EdgeDriver();
+
+       public boolean clickElement(By locator){
+        driver.findElement(locator).click();
+        return true;
+
+    }
+
+    //function 2 - isDisplayed
+    public boolean displayElement(By locator){
```

Comment:
By reading the name of the function, it looks like we are going to display the  element but in actual, we are checking if the element is displayed, could you please rename the function ?

Reply:
changed the name of the function to checkElementDisplay